Model/Category.php

```diff
@@ -270,7 +270,7 @@ public function getParent()
     /**
      * {@inheritdoc}
      */
-    public function setMedia(MediaInterface $media = null)
+    public function setMedia($media = null)
```

Comment:
If you remove type hinting, you should add a check in the setter for the type and explain why you did so in the comments